src/screens/ComponentScreen.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { cleanup, render } from '@testing-library/react-native'
+
+import { ComponentsScreen } from './ComponentsScreen'
+
+afterEach(cleanup)
```

Comment:
Trzeba sie dowiedziec co dokladnie robi `cleanup` i czy ten `afterEach` mialby byc wykonywany w kazdym pliku. Tutaj jest wiecej informacji - https://testing-library.com/docs/react-testing-library/setup/#skipping-auto-cleanup moze bedzie to jakos pomocne